app/next-client-app/components/update-SR-table/UpdateForm.tsx

```diff
@@ -17,9 +17,11 @@ export interface ShortFields {
 export function UpdateForm({
   scanreportFields,
   scanreportTable,
+  permissions,
 }: {
   scanreportFields: ShortFields[];
   scanreportTable: ScanReportTable;
+  permissions: string[];
```

Comment:
As before - should this be of type `Permission[]`

https://github.com/Health-Informatics-UoN/CaRROT-Mapper/blob/8d9b83936c438415caabc1073306261c50e57fdb/app/next-client-app/types/permissions.ts